static/app/data/platforms.tsx

```diff
@@ -1,47 +1,11 @@
 import integrationDocsPlatforms from 'integration-docs-platforms';
 import sortBy from 'lodash/sortBy';
 
-import {migratedDocs} from 'sentry/components/onboarding/gettingStartedDoc/sdkDocumentation';
 import {t} from 'sentry/locale';
 import {PlatformIntegration} from 'sentry/types';
 
 import {tracing} from './platformCategories';
 
-const migratedJavascriptPlatforms = {
-  id: 'javascript',
-  name: 'Browser JavaScript',
-  integrations: [
-    ...(integrationDocsPlatforms.platforms
-      .filter(platform => platform.id === 'javascript')?.[0]
-      ?.integrations?.filter(integration => !migratedDocs.includes(integration.id)) ??
-      []),
-    {
-      id: 'javascript-react',
-      link: 'https://docs.sentry.io/platforms/javascript/guides/react/',
-      name: 'React',
-      type: 'framework',
-    },
-    {
-      id: 'javascript-remix',
-      link: 'https://docs.sentry.io/platforms/javascript/guides/remix/',
-      name: 'Remix',
-      type: 'framework',
-    },
-    {
-      id: 'javascript-angular',
-      link: 'https://docs.sentry.io/platforms/javascript/guides/angular/',
-      name: 'Angular',
-      type: 'framework',
-    },
-    {
-      id: 'javascript-vue',
-      link: 'https://docs.sentry.io/platforms/javascript/guides/vue/',
-      name: 'Vue',
-      type: 'framework',
-    },
-  ],
-};
-
```

Comment:
as we are not removing the docs from the sentry-docs now, we can remove this code.